avm/res/fabric/capacity/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -20,6 +20,10 @@ param serviceShort string = 'fcwaf'
 @description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
 param namePrefix string = '#_namePrefix_#'
 
+@description('Required. Email address used by resource. This value is tenant-specific and must be stored in the CI Key Vault in a secret named \'CI-MySecret\'.')
```

Comment:
Please update description with the actual kv secrets name

Reply:
Fixed.